.travis.yml

```diff
@@ -1,18 +1,34 @@
-sudo: false
+dist: trusty
+sudo: required
 language: java
 cache:
   directories:
   - $HOME/.m2
+addons:
+  apt:
+    packages:
+      - openjdk-6-jdk
```

Comment:
Do we need this line? Shouldn't the JDK be installed by the JDK line below?

Reply:
Fully needed. not shipped by default. Just the env label stays, without this it fails. See the travis issues pasted in the pull message

Reply:
I didn't see the issue buried in the commit message. I still don't think this is correct, as it is installing this package for every single build...